src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1604,3 +1604,8 @@ Prevent Long URL From Breaking Out of Container
 .icon-info::before {
     content: '\e086';
 }
+
+.table {
```

Comment:
This doesn't look safe, i.e. all other elements with class `table` will also get styled like this which may not be what we want. Why not follow the suggestions in the issue report of #9115?

Reply:
Sure sir!. I will get it done accordingly. Thank you for suggesting changes. I will close this pull request and create clean PR.

Reply:
@wkurniawan07, Sir i tried out the changes as suggested in PR #9115 

before performing changes.
![before](https://user-images.githubusercontent.com/24544436/50107768-8f3d1280-0259-11e9-88a0-0f9a18bbf38b.png)
(code in inspect element is the code where the changes were suggested.) 

after performing the suggested changes.
![after1](https://user-images.githubusercontent.com/24544436/50107852-bd225700-0259-11e9-8a94-8a95e283af66.png)

The changes are able to get the rubric question tab below the pull down menu.

Reply:
@wkurniawan07  Sir, I carried out the changes as mentioned in PR #9115.

Rubric Question before changes
![before](https://user-images.githubusercontent.com/24544436/50108181-9fa1bd00-025a-11e9-943b-198c18409d8e.png)
(The code in the inspect element tab was suggested to be changed.)

Rubric Question after changes
![after1](https://user-images.githubusercontent.com/24544436/50108282-d7a90000-025a-11e9-9cfb-b98312c1aa49.png)
(The code in the inspect element is the modified code)

Please note: The problem of overflow still exits. Even if the table now is appearing under the pull down menu

Sir as correctly pointed out by you.
The  css code for the table will effect every class which has class name table.

Sir, but the effect of this css code is not observed on tables which are not overflowing. as the overflow only in x direction is set to auto and the display is of block nature.

.table{
display: block;
overflow-x: auto;
}

Sir, can you please suggest a better direction for solving this problem.

Thank you sir.